src/Simulations/run.jl

```diff
@@ -131,7 +140,12 @@ function time_step!(sim::Simulation)
 
     else # business as usual...
         Δt = aligned_time_step(sim, sim.Δt)
-        time_step!(sim.model, Δt, callbacks=model_callbacks)
+        if Δt > (sim.Δt / 1e10)
+            time_step!(sim.model, Δt, callbacks=model_callbacks)
+        else
+            println("Skipping aligned time step, which is of ", Δt)
+            sim.model.clock.time = next_actuation_time(sim)
```

Comment:
Could this also be

```suggestion
            sim.model.clock.time += Δt
```

?

Reply:
I don't think this works when we're _almost_ at `next_actuation_time` since it would skip that actuation time, no?

Reply:
What is

```julia
sim.model.clock.time + Δt
```

vs 

```julia
next_actuation_time(sim)
```

?

Reply:
@tomchor still have this question

Reply:
Ah, sorry I missed this. It's been so long that I forgot the details, but looking at the code for both, it's possible they end up calculating the same thing since `Δt` is calculated with `aligned_time_step()`.